build/build-image/Dockerfile

```diff
@@ -90,18 +91,14 @@ RUN echo "source <(helm completion bash)" >> /root/.bashrc
 #   |___/                                               |___/
 
 # install go-proto-gen 1.1
```

Comment:
Since we're here, should we update this comment as well? :smile:

Reply:
Might as well :-)

Reply:
Done.